packages/podge/podge.0.2/descr

```diff
@@ -0,0 +1,6 @@
+Shortcuts and helpers for common tasks in OCaml ecosystem
+
+If you're doing any modern OCaml then you're doubtlessly annoyed by
+the state of libraries and committing to one of the big ones can be
+restricting. Podge is a single modules containing specialized modules
```

Comment:
With git you can amend commits and then force push the branch, that would've been easiest in your case